src/test/java/org/openrewrite/java/migrate/guava/NoGuavaPrimitiveAsListTest.java

```diff
@@ -0,0 +1,243 @@
+              
+              class Test {
+                  List<Double> doubles = Doubles.asList(1d, 2d);
+              }
+              """,
+            """
+              import java.util.Arrays;
+              import java.util.List;
+
+              class Test {
+                  List<Double> doubles = Arrays.asList(1d, 2d);
+              }
+              """
+          )
+        );
+    }
+
+    @DocumentExample
+    @Test
+    void replaceFloat() {
+        //language=java
+        rewriteRun(
+          java(
+            """
+              import com.google.common.primitives.Floats;
+              import java.util.List;
+              
+              class Test {
+                  List<Float> floats = Floats.asList(1f, 2f);
+              }
+              """,
+            """
+              import java.util.Arrays;
+              import java.util.List;
+
+              class Test {
+                  List<Float> floats = Arrays.asList(1f, 2f);
+              }
+              """
+          )
+        );
+    }
+
+    @DocumentExample
+    @Test
+    void replaceLong() {
+        //language=java
+        rewriteRun(
+          java(
+            """
+              import com.google.common.primitives.Longs;
+              import java.util.List;
+              
+              class Test {
+                  List<Long> longs = Longs.asList(1L, 2L);
+              }
+              """,
+            """
+              import java.util.Arrays;
+              import java.util.List;
+
+              class Test {
+                  List<Long> longs = Arrays.asList(1L, 2L);
+              }
+              """
+          )
+        );
+    }
+
+    @DocumentExample
+    @Test
+    void replaceInt() {
+        //language=java
+        rewriteRun(
+          java(
+            """
+              import com.google.common.primitives.Ints;
+              import java.util.List;
+              
+              class Test {
+                  List<Integer> ints = Ints.asList(1, 2);
+              }
+              """,
+            """
+              import java.util.Arrays;
+              import java.util.List;
+
+              class Test {
+                  List<Integer> ints = Arrays.asList(1, 2);
+              }
+              """
+          )
+        );
+    }
+
+    @DocumentExample
+    @Test
+    void replaceShort() {
+        //language=java
+        rewriteRun(
+          java(
+            """
+              import com.google.common.primitives.Shorts;
+              import java.util.List;
+              
+              class Test {
+                  List<Short> shorts = Shorts.asList((short) 1, (short) 2);
+              }
+              """,
+            """
+              import java.util.Arrays;
+              import java.util.List;
+
+              class Test {
+                  List<Short> shorts = Arrays.asList((short) 1, (short) 2);
+              }
+              """
+          )
+        );
+    }
+
+    @DocumentExample
```

Comment:
We typically only have a single test as an example, as we previously showed these in the docs and having too many might clutter the docs; let's remove these from all but one representative example, which is then also the one to copy when folks report an issue.
```suggestion
```